src/entryPoints/AuthModule/Common/CommonAuthUtils.res

```diff
@@ -18,9 +18,17 @@ let passwordKeyValidation = (value, key, keyVal, errors) => {
       if !Js.Re.test_(%re("/^(?=.*[0-9])/"), value) {
         mustHave->Array.push("numeric")
       }
-      if !Js.Re.test_(%re("/^(?=.*[!@#$%^&*_])/"), value) {
+      if (
+        !Js.Re.test_(
+          Js.Re.fromString("^(?=.*[!@#$%^&*()_+\\-=\\[\\]{};':\"\\\\|,.<>\\/?`~])"),
```

Comment:
What are we checking in this - can add a comment so it's better what regex is doing

Reply:
and we can use Re directly from core api just check once

Reply:
Will add what all special characters are accepted in the description and for using Re directly from core api  will raise a separate PR for all to regex functions to core spi

Reply:
Great